app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
Ah yeah i thought about that, with two arguments its kinda a toss to me.  I'd still have to look at the method signature i think to make sure i was passing the right thing.  I'm good with either